src/vs/platform/extensions/node/extensionValidator.ts

```diff
@@ -219,7 +221,8 @@ export function isValidExtensionVersion(version: string, extensionDesc: IReduced
 		return true;
 	}
 
-	return isVersionValid(version, extensionDesc.engines.vscode, notices);
+	// {{SQL CARBON EDIT}}
+	return (extensionDesc.engines.sqlops && extensionDesc.engines.sqlops === '*') || isVersionValid(version, extensionDesc.engines.vscode, notices);
```

Comment:
Did you mean to leave `extensionDesk.engines.vscode` in the version check here? We want to check the sqlops version, right?

Reply:
no changed it to sqlops 

---
In reply to: [177604383](https://github.com/Microsoft/sqlopsstudio/pull/1011#discussion_r177604383) [](ancestors = 177604383)